src/main/java/homework_2/pyramid_printer/Main.java

```diff
@@ -0,0 +1,9 @@
+package homework_2.pyramid_printer;
+
+public class Main {
+
+    public static void main(String[] args) {
+        PyramidPrinter pyramidPrinter = new PyramidPrinter();
+        pyramidPrinter.run();
```

Comment:
Opt: you could use new PyramidPrinter().run();

Reply:
Yeah, you're right! What about HW-2 & HW-3 labels? Should I fix smth else?

Reply:
For now seems fine. I'll continue review when I have time